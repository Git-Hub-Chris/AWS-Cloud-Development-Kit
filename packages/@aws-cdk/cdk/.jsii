{
  "author": {
    "name": "Amazon Web Services",
    "organization": true,
    "roles": [
      "author"
    ],
    "url": "https://aws.amazon.com"
  },
  "description": "Deprecated: this module ha been renamed to @aws-cdk/core",
  "docs": {
    "deprecated": "This module has been renamed to @aws-cdk/core",
    "stability": "deprecated"
  },
  "homepage": "https://github.com/awslabs/aws-cdk",
  "jsiiVersion": "0.11.3 (build 464b8c6)",
  "license": "Apache-2.0",
  "name": "@aws-cdk/cdk",
  "readme": {
    "markdown": "This module has been renamed to @aws-cdk/core\n<!--BEGIN STABILITY BANNER-->\n\n---\n\n![Stability: Deprecated](https://img.shields.io/badge/stability-Deprecated-critical.svg?style=for-the-badge)\n\n> This API may emit warnings. Backward compatibility is not guaranteed.\n\n---\n<!--END STABILITY BANNER-->\n\n"
  },
  "repository": {
    "directory": "packages/@aws-cdk/cdk",
    "type": "git",
    "url": "https://github.com/awslabs/aws-cdk.git"
  },
  "schema": "jsii/0.10.0",
  "targets": {
    "java": {
      "maven": {
        "artifactId": "cdk",
        "groupId": "software.amazon.awscdk"
      },
      "package": "software.amazon.awscdk"
    },
    "js": {
      "npm": "@aws-cdk/cdk"
    },
    "python": {
      "distName": "aws-cdk.cdk",
      "module": "aws_cdk.cdk"
    }
  },
  "types": {},
  "version": "0.35.0",
  "fingerprint": "ia0gOKzBkyR6GiGt0pvF9/ZEsI2wa9dGr/PZyezy1zI="
}
