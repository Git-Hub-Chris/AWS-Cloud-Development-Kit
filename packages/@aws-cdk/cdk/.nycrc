/Users/benisrae/code/cdk/aws-cdk-2/tools/cdk-build-tools/config/nycrc