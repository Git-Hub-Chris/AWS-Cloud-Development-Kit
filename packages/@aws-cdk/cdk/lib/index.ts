export * from './aspect';
export * from './tag-aspect';

export * from './construct';
export * from './token';
export * from './unresolved';
export * from './tag-manager';
export * from './dependency';

export * from './cloudformation/cloudformation-json';
export * from './cloudformation/reference';
export * from './cloudformation/cfn-condition';
export * from './cloudformation/fn';
export * from './cloudformation/include';
export * from './cloudformation/logical-id';
export * from './cloudformation/cfn-mapping';
export * from './cloudformation/cfn-output';
export * from './cloudformation/cfn-parameter';
export * from './cloudformation/pseudo';
export * from './cloudformation/cfn-resource';
export * from './cloudformation/resource-policy';
export * from './cloudformation/cfn-rule';
export * from './cloudformation/stack';
export * from './cloudformation/cfn-element';
export * from './cloudformation/dynamic-reference';
export * from './cloudformation/tag';
export * from './cloudformation/removal-policy';
export * from './cloudformation/arn';
export * from './cloudformation/secret';

export * from './app';
export * from './context';
export * from './environment';

export * from './runtime';

export * from './synthesis';
