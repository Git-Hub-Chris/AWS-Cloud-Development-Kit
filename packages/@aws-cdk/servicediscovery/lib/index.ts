// The L1 Library for AWS::ServiceDiscovery:
export * from '../cfn/servicediscovery';
