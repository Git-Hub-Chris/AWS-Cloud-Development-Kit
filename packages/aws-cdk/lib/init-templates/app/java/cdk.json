{
  "app": "mvn compile -q && mvn -e -q exec:java"
}
