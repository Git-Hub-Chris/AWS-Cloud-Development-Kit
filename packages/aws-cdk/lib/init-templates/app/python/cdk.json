{
    "app": "python3 hello/hello_stack.py"
}
