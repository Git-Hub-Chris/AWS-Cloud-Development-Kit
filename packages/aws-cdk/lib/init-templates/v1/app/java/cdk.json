{
  "app": "mvn -e -q compile exec:java",
  "watch": {
    "exclude": [
      "target",
      "src/test",
      "pom.xml",
      "README.md",
      "cdk.json"
    ]
  }
}
