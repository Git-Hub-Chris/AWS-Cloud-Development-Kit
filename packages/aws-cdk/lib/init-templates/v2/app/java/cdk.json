{
  "app": "mvn -e -q compile exec:java",
  "watch": {
    "include": ["**"],
    "exclude": [
      "target",
      "src/test",
      "pom.xml",
      "README.md",
      "cdk.json"
    ]
  }
}
