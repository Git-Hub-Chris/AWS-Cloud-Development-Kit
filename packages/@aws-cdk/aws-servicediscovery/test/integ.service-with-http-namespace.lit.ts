import cdk = require('@aws-cdk/cdk');
import servicediscovery = require('../lib');

const app = new cdk.App();
const stack = new cdk.Stack(app, 'aws-servicediscovery-integ');

const namespace = new servicediscovery.HttpNamespace(stack, 'MyNamespace', {
  name: 'covfefe',
});

const service = namespace.createService('Service', {
  description: 'service description',
});

service.registerNonIpInstance({
  customAttributes: { arn: 'arn:aws:s3:::mybucket' }
});

app.run();
