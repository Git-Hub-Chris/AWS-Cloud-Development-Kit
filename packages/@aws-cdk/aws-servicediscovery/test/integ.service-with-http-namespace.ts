import cdk = require('@aws-cdk/cdk');
import servicediscovery = require('../lib');

const app = new cdk.App();
const stack = new cdk.Stack(app, 'aws-servicediscovery-integ');

const namespace = new servicediscovery.HttpNamespace(stack, 'MyNamespace', {
  name: 'covfefe',
});

new servicediscovery.Service(stack, 'Service', {
  name: 'service',
  namespace,
  description: 'service description',
  healthCheckCustomConfig: {
    failureThreshold: 3,
  }
});

app.run();
