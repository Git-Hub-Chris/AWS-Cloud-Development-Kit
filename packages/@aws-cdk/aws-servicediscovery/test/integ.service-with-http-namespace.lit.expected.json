{
  "Resources": {
    "MyNamespaceD0BB8558": {
      "Type": "AWS::ServiceDiscovery::HttpNamespace",
      "Properties": {
        "Name": "covfefe"
      }
    },
    "MyNamespaceService6E3133E7": {
      "Type": "AWS::ServiceDiscovery::Service",
      "Properties": {
        "Description": "service description",
        "NamespaceId": {
          "Fn::GetAtt": [
            "MyNamespaceD0BB8558",
            "Id"
          ]
        }
      }
    },
    "MyNamespaceServiceNonIpInstance59424747": {
      "Type": "AWS::ServiceDiscovery::Instance",
      "Properties": {
        "InstanceAttributes": {
          "arn": "arn:aws:s3:::mybucket"
        },
        "ServiceId": {
          "Fn::GetAtt": [
            "MyNamespaceService6E3133E7",
            "Id"
          ]
        },
        "InstanceId": "awsservicediscoveryintegMyNamespaceServiceNonIpInstance360FCE8A"
      }
    }
  }
}